### DISCUSSION TITLE
STM32 Nucleo-H563ZI Board not recognized

### DISCUSSION THREAD

**Discussion by DF4OR** [stlink-org/stlink#1381] [Q&A (resolved)] [answered]
Hi.
Beginner here. 
I have sucessfully used other boards (e.g. Nucleo-F091RC) with the current st-link utils on Linux. The Nucleo-H563ZI board in contrast is not properly recognized and cannot be programmed.

Output of lsusb 
`Bus 003 Device 017: ID 0483:374e STMicroelectronics STLINK-V3`

Output of st-info --probe
```Found 1 stlink programmers
  version:    V3J10
  serial:     003A001B3132511138363431
  flash:      0 (pagesize: 0)
  sram:       0
  chipid:     0x000
  dev-type:   unknown
```
Outout of st-info --version
`v1.8.0`

System
* Arch Linux, kernel 6.8.1-arch1-1
* stlink utils installed with pacman

Any idea what's going on?

TIA,
Ekki

**Comment by Nightwalker-87** [comment, marked as answer]
I can verify this with the same hardware.
As far as I can see, support for this board/chip has not been (fully) implemented yet.

**Comment by Nightwalker-87** [comment]
Follow up in issue #1384.
